ethereum/eth/src/main/java/org/hyperledger/besu/ethereum/eth/manager/EthProtocolManager.java

```diff
@@ -380,11 +380,12 @@ public void handleNewConnection(final PeerConnection connection) {
   @Override
   public boolean shouldConnect(final Peer peer, final boolean incoming) {
     if (peer.getForkId().map(forkId -> forkIdManager.peerCheck(forkId)).orElse(true)) {
-      LOG.trace("ForkId OK or not available");
+      LOG.debug("ForkId OK or not available for peer {}", peer.getId());
       if (ethPeers.shouldConnect(peer, incoming)) {
         return true;
       }
     }
+    LOG.debug("ForkId check failed for peer {}", peer.getId());
```

Comment:
I think slightly more efficient to use LOG.atDebug() here

Reply:
done